samples/sample_multi_transcode/src/transcode_utils.cpp

```diff
@@ -2119,28 +2239,26 @@ mfxStatus CmdProcessor::VerifyAndCorrectInputParams(TranscodingSample::sInputPar
     }
 
     // valid target usage range is: [MFX_TARGETUSAGE_BEST_QUALITY .. MFX_TARGETUSAGE_BEST_SPEED] (at the moment [1..7])
-    if ((InputParams.nTargetUsage < MFX_TARGETUSAGE_BEST_QUALITY) ||
+    // If target usage is kept unknown - presets manager will fill in proper value
+    if ((InputParams.nTargetUsage < MFX_TARGETUSAGE_UNKNOWN) ||
         (InputParams.nTargetUsage > MFX_TARGETUSAGE_BEST_SPEED) )
     {
-        if (InputParams.nTargetUsage == MFX_TARGETUSAGE_UNKNOWN)
-        {
-            // if user did not specified target usage - use balanced
-            InputParams.nTargetUsage = MFX_TARGETUSAGE_BALANCED;
-        }
-        else
-        {
-            PrintError(NULL,"Unsupported target usage");
-            // report error if unsupported target usage was set
+            PrintError((const msdk_char*)_T("Unsupported target usage"));
```

Comment:
Causes build to fail on Linux. Suggest replacing
    PrintError((const msdk_char*)_T("Unsupported target usage"));
with
     PrintError(MSDK_STRING("Unsupported target usage"));

Reply:
Fixed.